src/VisualStudio/IntegrationTest/IntegrationTests/CSharp/CSharpGenerateTypeDialog.cs

```diff
@@ -43,7 +43,7 @@ void Method()
             GenerateTypeDialog.VerifyClosed();
         }
 
-        [WpfFact, Trait(Traits.Feature, Traits.Features.CodeActionsGenerateType)]
+        [WpfFact(Skip= "https://github.com/dotnet/roslyn/issues/26205"), Trait(Traits.Feature, Traits.Features.CodeActionsGenerateType)]
```

Comment:
❗️ This one should be fixed rather than skipped. The underlying cause is #26039 which describes the problem and the fix.

Reply:
@sharwell At this point, if the issue is understood then we should just disable it. There's no reason to be inflicting pain on other people trying to get unrelated changes in.

Reply:
Agree. We should disable unless there is actually a fix in progress.